great_expectations/data_context/types/base.py

```diff
@@ -1259,7 +1259,7 @@ def to_json_dict(self):
         return {
             "base_url": self.base_url,
             "organization_id": self.organization_id,
-            "access_token": self.access_token,
+            "access_token": "xxx",  # self.access_token,
```

Comment:
@joshua-stauffer Could we please pick a different string?  For example, "abc".  Thanks!

Reply:
hi @alexsherstinsky -- sure, I'll update this to match `PasswordMasker.MASKED_PASSWORD_STRING`. Thanks for the feedback!